engine/release-notes/18.09.md

```diff
@@ -5,6 +5,14 @@ toc_max: 2
 skip_read_time: true
 ---
 
+> **Note:**
+>
+> The client and container runtime are now in separate packages from the daemon
+> in Docker Engine 18.09. Users should install and update all three packages at
+> the same time to get the latest patch releases. For example, on Ubuntu:
```

Comment:
```suggestion
> With this release, the daemon, client and container runtime are now all shipped in separate packages. So you > need to install and later update all three packages at the same time to get the latest patch releases. For example, in Ubuntu:
```

Reply:
I think we have that covered in the install page; for a "fresh" install, there should not be a need to specify all packages (they will be installed as dependency), so it's mostly for updating packages to force them being updated, but we mention those steps;


<img width="982" alt="Screenshot 2022-05-26 at 00 20 39" src="https://user-images.githubusercontent.com/1804568/170378570-92fdfb57-c2cc-4140-bc7f-9781bf1f1a1e.png">
<img width="718" alt="Screenshot 2022-05-26 at 00 21 04" src="https://user-images.githubusercontent.com/1804568/170378577-8995db16-bffd-4040-86e8-e35df484f0c3.png">

Reply:
Yes. We are telling users to remove all the packages ( to remove older installations ) in the instruction but the nexus behind the change (for some users at least) in number/structure of packages is the bit I thought was interesting mentioning here in addition to in the release notes. Although it's bean a while (18.xx) maybe it is still interesting.

Reply:
Yeah, I don't think many people will arrive at the 18.09 release notes; I mostly wanted to preserve the information (instead of removing it altogether).